frontend/mulighetsrommet-veileder-flate/src/components/filtrering/Filtermeny.tsx

```diff
@@ -22,17 +23,26 @@ const Filtermeny = () => {
         Filter
       </Heading>
       <Searchfield sokefilter={filter.search!} setSokefilter={(search: string) => setFilter({ ...filter, search })} />
-      <CheckboxFilter
-        accordionNavn="Innsatsgrupper"
-        options={filter.innsatsgrupper!}
-        setOptions={innsatsgrupper => setFilter({ ...filter, innsatsgrupper })}
+      <InnsatsgruppeFilter
+        accordionNavn="Innsatsgruppe"
+        option={filter.innsatsgruppe?.nokkel}
+        key={filter.innsatsgruppe?.nokkel}
+        setOption={innsatsgruppe => {
+          const foundInnsatsgruppe = innsatsgrupper.data?.find(gruppe => gruppe.nokkel === innsatsgruppe);
+          if (foundInnsatsgruppe) {
+            setFilter({
+              ...filter,
+              innsatsgruppe: {
+                id: foundInnsatsgruppe?._id,
+                tittel: foundInnsatsgruppe?.tittel,
+                nokkel: foundInnsatsgruppe?.nokkel,
+              },
+            });
```

Comment:
Radioknappene holder på verdien til `innsatsgruppe.nokkel`. Vi trenger å hente ut resten av objektet til innsatsgruppen når man trykker på en radioknapp for innsatsgruppe, så her søker vi blant innsatsgruppene fra Sanity-dataene, og setter korrekt innsatsgruppe basert på nøkkelen.